src/generic/datavgen.cpp

```diff
@@ -3192,6 +3356,303 @@ int wxDataViewMainWindow::GetLineHeight( unsigned int row ) const
 }
 
 
+void RowRange::Add(const unsigned int idx)
+{
+    RangeEntity *rng = NULL;
```

Comment:
Raw pointers should be avoided in C++ code, instead of declaring it here you should just use a reference below:

    RangeEntity& rng = m_ranges[rngIdx];